src/LibraryManager.Vsix/Json/TextviewCreationListener.cs

```diff
@@ -89,7 +98,7 @@ public void VsTextViewCreated(IVsTextView textViewAdapter)
             return files;
         }
 
-        private async Task<IEnumerable<string>> GetFilesAsync(ILibraryInstallationState state)
+        private async Task<IEnumerable<string>> GetFilesWithVersionsAsync(ILibraryInstallationState state)
         {
             if (state.Files == null)
```

Comment:
If state has files (the case where you specify the files you want from a library in the manifest), then we won't append the version causing failures.